File: 프로그래머스/unrated/181833. 특별한 이차원 배열 1/특별한 이차원 배열 1.java
class Solution {
    public int[][] solution(int n) {
        int[][] result = new int[n][n];
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < n; j++) {
                if (i == j) {
                    result[i][j] = 1;
                    continue;
                }
                result[i][j] = 0;
            }
        }
        return result;
    }
}

File: 프로그래머스/unrated/181833. 특별한 이차원 배열 1/특별한 이차원 배열 1.js
function solution(n) {
    let result = Array.from(Array(n), () => new Array(n));
    for (let i = 0; i < n; i++) {
        for (let j = 0; j < n; j++) {
            if (i === j) {
                result[i][j] = 1;
                continue;
            }
            result[i][j] = 0;
        }
    }
    return result;
}